### ISSUE TITLE
Spurious warnings in f-strings with Python 3.12

### ISSUE THREAD

**Issue by bkeryan** [ni/python-styleguide#142]
With Python 3.12, `ni-python-styleguide` starts emitting E713 for this code:
```python
        if not service:
            raise RuntimeError(
                f"Service class '{service_class}' not found in '{service_config_file}'"
            )
```
```
.\ni_measurementlink_service\measurement\service.py:230:51: E713 test for membership should be 'not in'
```

This is because `ni-python-styleguide` still depends on `pycodestyle` 2.9.1:
- `flake8 = "^5.0"` --> 5.0.4
- `pycodestyle>=2.9.0,<2.10.0` --> 2.9.1

`pycodestyle` 2.11 includes multiple updates for f-string handling. I cannot reproduce this problem with `pycodestyle` main.

**Comment by mshafer-NI** [comment]
Would you like to raise a PR to use newer flake8 and pycodestyle for 3.11+?
